tests/CssToInlineStylesTest.php

```diff
@@ -200,10 +200,32 @@ public function testInvalidSelector()
 
     public function testHtmlEncoding()
     {
-        $text = 'Žluťoučký kůň pije pivo nebo jak to je dál';
-        $expectedText = '&#x17D;lu&#x165;ou&#x10D;k&#xFD; k&#x16F;&#x148; pije pivo nebo jak to je d&#xE1;l';
+        $text
+            = 'Žluťoučký kůň pije pivo nebo jak to je dál @ €';
+        $expectedText
+            = 'Žluťoučký kůň pije pivo nebo jak to je dál @ €';
 
-        $this->assertEquals($expectedText, trim(strip_tags($this->cssToInlineStyles->convert($text, ''))));
+        $this->assertEquals($expectedText, trim(strip_tags($this->cssToInlineStyles->convert($text))));
+    }
+
+    public function testSpecialCharacters()
+    {
+        $text
+            = '1 &lt; 2';
```

Comment:
well the reason is to be human readable so you can see that two lines are the same
Or would you prefer to have only one variable for $actual and $expected?